md/ui-designer-overview.md

```diff
@@ -80,6 +80,15 @@ In the Subscription editions, you can [update a form in production](live-update.
 
 To use a layout in a business application, export it from the Bonita UI designer, import it as a Bonita Portal resource, and map it as the layout of the relevant application.
 
+### Business Data model Synchronization
+
+Business Data Model is synchronized from Bonita Studio to Bonita UI-Designer when you deployed your BDM. 
+This synchronization is done by a new independent process and managed by Bonita Studio.
+
+See [Data management](variables.md#business-data) section to know how to use this Business Data Model.
+
+If an error occurs, restart you Bonita Studio allow you restart this process.
```

Comment:
Finally, I think we should say this here. It is more a trouble shooting stuff, that should not appear in the general section.
Not sure we have a trouble shooting section, so the icon+tooltip in the UID should be enough.